grpcutil/jwt_auth.go

```diff
@@ -108,3 +125,45 @@ func (g *JWTAuthenticationHandler) RequestPrincipal(ctx context.Context) (string
 
 	return principal, nil
 }
+
+type HandlerOption func(handler *JWTAuthenticationHandler) *JWTAuthenticationHandler
+
+// NoValidation disables certificate validation for JWT.
+func NoValidation() HandlerOption {
```

Comment:
Similarly, I recommend add `JWTAuth` as a substring for these functions. E.g. `NoJWTAuthValidation`, `WithJWTAuthEndpoint`. This would help avoid future naming conflicts.